### DISCUSSION TITLE
Socket does not start

### DISCUSSION THREAD

**Discussion by AnonymousMhl** [DNSCrypt/dnscrypt-proxy#2718] [Q&A]
Hi, despite trying to configure it and repeat the installation several times, "Dnscryp-proxy.socket" does not want to start correctly, I can't understand the reason:
![Screenshot_2024-10-29_03-46-56](https://github.com/user-attachments/assets/ba010bb5-2b79-48c8-b743-47ec9b35cef6)

`
sudo systemctl status dnscrypt-proxy.service
`
![2](https://github.com/user-attachments/assets/cf317000-5fcc-498f-a2bc-af8e4c7e8ad3)

**Comment by AnonymousMhl** [comment]
I haven't managed to get the daemon to work yet

**Comment by lifenjoiner** [comment]
What is the result of `/usr/sbin/dnscrypt-proxy -config /etc/dnscrypt-proxy/dnscrypt-proxy.toml`?

v2.0.45 is very old, maybe out of maintenance.
I saw "complete the setup with a reboot" from https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-on-Debian-and-Ubuntu.
And maybe you can try the generic Linux installation procedure manually.
